doc/installing.rst

```diff
@@ -6,11 +6,160 @@ Installing PYPIT
+
+Linetools can be installed from github or using pip. 
+To install a development version of linetools from the source::
+
+	mkdir linetools
+	cd linetools
+	git clone https://github.com/linetools/linetools.git
+	python setup.py develop
+
+To install linetools using pip::
+
+	pip install linetools
+
+According to the linetools documentation page, "If you wish to have full functionality of the GUIs and are using MacOSX, then you probably need to change your backend from macosx to TkAgg in the matplotlibrc file."
+
+If you'd like to make sure that linetools has been properly installed, you can run the following test::
+
+	python -c 'import linetools; linetools.test()'
+
+If the test reports no errors, then linetools was installed without issue. 
+
+GSL
+---
+
+The package complies Cython code that links to gsl routines.
+These must be installed on your system prior to PYPIT installation.
+We recommend that if you need to install GSL that you use Anaconda,
+e.g.::
+
+    conda install -c https://conda.anaconda.org/asmeurer gsl
+
+Beware:  multiple installations of libgsl is likely to cause a problem..
+
+You are also required to point the ENVIRONMENTAL varible
+GSL_PATH to the path above the lib/ and include/ files.
+
+You should be able to determine that path with::
+
+    gsl-config --prefix
+
+In my case, anaconda installs packages in /Users/USERNAME/anaconda 
+
+You then need to edit your .bashrc, .tcshrc, or .cshrc file with this path.
+If you are using bash (and the path mentioned above as an example)::
+
+	export GSL_PATH='/Users/USERNAME/anaconda'
+
+If instead you are using tcsh or csh::
+
+	setenv GSL_PATH "/Users/USERNAME/anaconda"
+
+Installing PYPIT
+================
+
+We recommend that you grab the code from github::
+
+    mkdir PYPIT
+	cd PYPIT
+	git clone https://github.com/PYPIT/PYPIT.git
+
+From there, you can build and install either with install or develop, e.g.::
+
+    python setup.py develop
+
+or::
+
+	python setup.py install
+
+This should compile all the necessary Cython files, etc.
+
+**If you will be editing/contributing to the code:** 
+
+Note that the default branch you will be in once you clone PYPIT is the master branch. You may want to switch to a different branch by using a command like::
+
+	 git checkout OTHERBRANCHNAME
+
+Check which branch you are in with the command::
+
+	git branch
+
+(an asterisk will appear next to the branch that is currently checked out)
+
+Tests
+=====
+In order to assess whether PYPIT has been properly installed, we suggest you run the following tests:
+
+1. Ensure run_pypit works
+----------------------------
+Go to a directory outside of the PYPIT directory (e.g. your home directory), then type run_pypit. 
+::
+	cd
+	run_pypit
+
+Note: This may not work if you are checked into a branch that is not the master branch (e.g. new_docs). 
+
+If this step is failing, try::
+
+	git checkout master
+
+Then try again. 
+
+
+2. Try the test suite
+---------------------
+We 
```

Comment:
Let's differentiate between the unit tests (which are run
with setup.py) and the test suite which is on actual data files.